docs/60_initialize_vendors.md

```diff
@@ -221,7 +221,7 @@ You'll be prompted with the following form -- see the example values populated i
 
 <br/><br/>
 <img
-src="../img/register_new.jpg"
+src="../img/register_new_oauth.jpg"
```

Comment:
Should we remove the `register_new.jpg` file?  It doesn't seem to be used elsewhere.

Reply:
ah good catch -- removed!